### PR TITLE
#787: Added submission views documentation

### LINKED ISSUE #787: Submission Views documentation
Either incorporating into existing [Views](https://docs.synapse.org/articles/views.html) article, or a new standalone article (which might be justified since the rows do not represent entities, and the scope definition will be so different).

### PR DESCRIPTION
**Pull request by marcomarasca**
Added the submission views documentation to the evaluation queues article. Includes how to create a submission view and how to embed it into a wiki page. Additionally added the reference to the submission views to the views article.

- [x] Fixes #787 (Link Github Issue if appropriate?)
- [ ] If you are creating a page, did you follow the instructions [here](https://github.com/Sage-Bionetworks/synapseDocs#creating-a-page)?
- [x] Did you follow the [style guide](https://github.com/Sage-Bionetworks/synapseDocs#style-guide)?
testing-your-github-pages-site-locally-with-jekyll))?
- [x] Did you include a description of your changes or additions in this pull request?
- [ ] (Optional) Did you examine your build locally (instructions [here](https://help.github.com/en/github/working-with-github-pages/))

### CODE REVIEW

File: _articles/evaluation_queues.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To create a queue, you must first create a Synapse `Project` or have [edit permi
 
 Once you've created your project, navigate to it and add `/challenge` to the url (e.g. www.synapse.org/#!Synapse:syn12345/challenge). Click **Tools** in the right corner and select **Add Evaluation Queue**.
 
-<img style="width: 80%;" src="/assets/images/create_evaluation_queues.png">
+<img style="width: 40%;" src="../assets/images/create_evaluation_queues.png">
```
Comment (thomasyu888) [inline _articles/evaluation_queues.md:17]: Not sure you will need the `../`

File: _articles/evaluation_queues.md [hunk 2]
```diff
@@ -111,55 +111,55 @@ Every submission you make to an Evaluation queue has a unique ID. This ID should
...
 ## View Submissions of an Evaluation Queue
 
-Submissions can be viewed through leaderboard widgets on Wiki pages.
+Submissions can be viewed and shared with users through [Submission Views]({{ site.baseurl }}{% link _articles/views.md %#create-a-submission-view}) creating dynamic leaderboards. Submission annotations can be added to a SubmissionStatus object and are automatically indexed in the view.
 
-### Adding Leaderboard Widget
+### Creating the Submission View
 
-<img style="width: 80%;" src="/assets/images/add_leaderboard_widget.png">
+Navigate to the **Tables** tab and under the **Table Tools** menu in the upper right-hand select **Add Submission View**:
```
Comment (thomasyu888) [inline _articles/evaluation_queues.md:122]: Is this still an alpha mode feature?

**Review by thomasyu888 — APPROVED** [review]

**Comment by thomasyu888** [comment]
The content looks good.  May want to wait if this is an alpha mode feature.

File: _articles/evaluation_queues.md (hunk 2 above)
Comment (marcomarasca) [reply _articles/evaluation_queues.md:122]: Yep, I noticed that! I'll create a JIRA to take it out of alpha mode.

File: _articles/evaluation_queues.md (hunk 1 above)
Comment (marcomarasca) [reply _articles/evaluation_queues.md:17]: It's not strictly needed, but it allowed me to see the image in a markdown preview (without running a server)

**Comment by marcomarasca** [comment]
Thanks, yes makes sense to wait before merging till it's out of alpha mode.

**Comment by thomasyu888** [comment]
@marcomarasca , is the feature now out of alpha mode?

**Comment by marcomarasca** [comment]
> @marcomarasca , is the feature now out of alpha mode?

I think it is: I defer to @jay-hodgson on this :)

**Comment by thomasyu888** [comment]
@kelshmo , @stacey-w-taylor ,  This PR can be merged after this week's update to Synapse.  Submission Views are being moved out of Alpha mode.

**Comment by stacey-w-taylor** [comment]
> @kelshmo , @stacey-w-taylor , This PR can be merged after this week's update to Synapse. Submission Views are being moved out of Alpha mode.

@thomasyu888 Thanks for the update! I will merge next week.

File: _articles/challenge_administration.md [hunk 3]
```diff
@@ -88,6 +88,8 @@ Throughout the challenge, participants will continuously submit to the evaluatio
 
 ### Revealing Submissions and Scores
 
-Organizers can create a leaderboard when scores are ready to be revealed to participants. Leaderboards are sorted, paginated, tabular forms that display submission annotations (e.g. scores from the scoring application and other metadata) and update as annotations or scores change. A leaderboard can provide real-time insight into the progress of a challenge.
+Organizers can create leaderboards when scores are ready to be revealed to participants using [Submission Views]({{ site.baseurl }}{% link _articles/views.md %#create-a-submission-view}).
```
Comment (stacey-w-taylor) [inline _articles/challenge_administration.md:91]: @marcomarasca Could you verify that the syntax on this tag is correct? I am attempting to build the site locally to test your edits/additions, and I am getting an error for this link. I am not a liquid expert (yet) so I can't spot the issue. Thanks!

Comment (marcomarasca) [reply _articles/challenge_administration.md:91]: Thanks for spotting this @stacey-w-taylor, I think there is a typo. Will double check this!

Comment (marcomarasca) [reply _articles/challenge_administration.md]: Fixed the anchor link!

**Review by stacey-w-taylor — APPROVED** [review]

**Pull request merged**